netaddr.go

```diff
@@ -158,10 +158,12 @@ func (ip IP) Less(ip2 IP) bool {
 	switch bytes.Compare(a16[:], b16[:]) {
 	case -1:
 		return true
-	default:
+	case 0:
```

Comment:
what was wrong with this?

Reply:
See https://github.com/dominikh/go-tools/issues/396. It seems as a weird check to me as well.

Reply:
This was slightly surprising to me as well. I think on the whole that staticcheck makes good suggestions and we could add a directive to ignore this one though.

Although the behavior of bytes.Compare is guaranteed by the stdlib, I've typically taken a stance of using panics in default cases for impossible behavior like `bytes.Compare(a, b) == 2`.

Reply:
I'd rather not have a panic case here on principle. Plus it's slightly faster to not have a 4th case. (probably barely measurable, but...)

But I guess we can arrange it. Just add a comment:

```
    default: // case 0 (bytes.Compare only returns -1, 1, 0)
```

Reply:
Sure, done.